tests/core/test_core_memory_resources.cpp

```diff
@@ -165,13 +168,15 @@ INSTANTIATE_TEST_SUITE_P(core_memory_resource_tests, core_memory_resource_test,
                              {{&host_resource, "host_resource"},
                               {&binary_resource, "binary_resource"},
                               {&contiguous_resource, "contiguous_resource"},
-                              {&arena_resource, "arena_resource"}}));
-
-INSTANTIATE_TEST_SUITE_P(core_memory_resource_stress_tests,
-                         core_memory_resource_stress_test,
-                         testing::Values(&host_resource, &binary_resource,
-                                         &arena_resource),
-                         vecmem::testing::memory_resource_name_gen(
-                             {{&host_resource, "host_resource"},
-                              {&binary_resource, "binary_resource"},
-                              {&arena_resource, "arena_resource"}}));
+                              {&arena_resource, "arena_resource"},
+                              {&instrumenting_resource,
+                               "instrumenting_resource"}}));
+
+INSTANTIATE_TEST_SUITE_P(
+    core_memory_resource_stress_tests, core_memory_resource_stress_test,
+    testing::Values(&host_resource, &binary_resource, &arena_resource),
+    vecmem::testing::memory_resource_name_gen(
+        {{&host_resource, "host_resource"},
+         {&binary_resource, "binary_resource"},
+         {&arena_resource, "arena_resource"},
+         {&instrumenting_resource, "instrumenting_resource"}}));
```

Comment:
I made this same mistake in the past myself as well. :stuck_out_tongue:

You need to list the new memory resource not just to `vecmem::testing::memory_resource_name_gen`, but more importantly to the `testing::Values` object. In the current setup the test would know how to name the resource, but it is not set up to actually use it. :smile: